HTML/mockup-product-basic/index.html

```diff
@@ -0,0 +1,184 @@
+                    $499
+                </div>
+                <div class="description__store-details">
+                    <p>
+                        Availability : <span class="u-sea-color">In stock</span>
+                    </p>
+                    <p>
+                        Product Code : <span class="u-sea-color">#4657</span>
+                    </p>
+                    <p>
+                        Tags : <span class="u-sea-color">Fashion, Hood, Classic</span>
+                    </p>
+                </div>
+                <div class="description__material-details">
+                    <p>
+                        Sleek, polished, and boasting an impeccably modern fit, this blue, 2-but-
+                        ton Lazio suit features a notch lapel, flap pockets, and accompanying flat
+                        front trousers—all in pure wool by Vitale Barberis Canonico.
+                    </p>
+                    <ul>
+                        <li>Dark blue suit for a tone-on-tone look</li>
+                        <li>Regular fit</li>
+                        <li>100% Cotton</li>
+                        <li>Free shipping with 4 days delivery</li>
+                    </ul>
+                </div>
+                <form class="description__selection">
+                    <div class="selection__color">
+                        <label for="color">color</label>
+                        <select id="color" class="selection__select-tag">
+                            <option disabled selected>Select Color </option>
+                            <option>Red</option>
+                            <option>Blue</option>
+                            <option>Rainbow</option>
+                        </select>
+                    </div>
+                    <div class="selection__size">
+                        <label for="size">size </label>
+                        <select id="size" class="selection__select-tag">
+                            <option disabled selected>Select size </option>
+                            <option>s</option>
+                            <option>m</option>
+                            <option>l</option>
+                        </select>
+                    </div>
+                    <div class="selection__quantity">
+                        <label for="quantity">qty</label>
+                        <input type="number" min=0 max=10 placeholder="1" id="quantity" class="selection__input">
+                    </div>
+                    <button class="selection__btn selection__btn--clear-btn">Clear Selection</button>
+                    <button class="selection__btn selection__btn--add-btn">add to cart</button>
+                    <!-- icon below come from https://fontawesome.com
+	                https://fontawesome.com/license-->
+                    <button class="selection__btn selection__btn--add-btn u-btn-white"><img
+                            src="img/heart-regular.svg">add to wishlist</button>
```

Comment:
it would looks more readable
```
<button class="selection__btn selection__btn--add-btn u-btn-white">
    <img src="img/heart-regular.svg">
    add to wishlist
</button>
```